doc/nrf/examples.rst

```diff
@@ -9,6 +9,10 @@ Documentation for those is available in the :ref:`zephyr:samples-and-demos` sect
 Nordic Semiconductor provides additional examples that specifically target Nordic Semiconductor devices and show how to implement typical use cases with our libraries and drivers.
 Samples showcase a single feature or library, while applications include a variety of libraries to implement a specific use case.
 
+.. note::
+   The default behavior in case of fatal error is system reset.
```

Comment:
Should we be a bit more detailed here? How about:

```
All samples and applications in the |NCS| are configured to perform a system reset if a fatal error occurs.
This behavior is different from how fatal errors are handled in the Zephyr samples.
You can change the default behavior by updating the configuration option :option:`RESET_ON_FATAL_ERROR`.
```

Reply:
Good point, though already there is an exception: asset_tracker.

Reply:
How about nRF Desktop?
If that uses it, I'm okay with changing to "Most" instead of "All". Otherwise we should move that note to the samples page instead of "Samples and applications".

Reply:
nRF Desktop is using the default configuration - so will reset the chip in case of fatal error.

Reply:
As discussed with @KAGA164 , I would not want the feature to be enabled in debug builds. In case of any crash I would like us or clients to be able to attach a debugger.
In case of release build it can be enabled but for us it makes a little difference as we will be rebooted by watchdog anyway.

Reply:
Sounds good. Should I remove the `.. note::` sentence

Reply:
No, keep the note. :)

Reply:
I changed text but I removed applications word.

Reply:
@pdunaj is it enough for you to have it like this `default y if !DEBUG && !KERNEL_DEBUG` ?